src/client/app/app.react.js

```diff
@@ -94,7 +96,7 @@ class App extends Component {
         <RouteHandler {...this.state} />
         <footer>
           <p>
-            made by <a href="https://twitter.com/steida">steida</a>
+            <FormattedHTMLMessage message={msg('app.madeByHtml')} />
```

Comment:
I also use it, but I don't think that's the best solution we can can come up with. Related: https://github.com/Kdyby/Translation/issues/4